http-gateway/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN cd /web && \
     npm config set fetch-retry-mintimeout 600000 &&  \
     npm config set fetch-retry-maxtimeout 1200000 && \
     npm config set fetch-timeout 1800000 && \
-    npm install && \
```

Comment:
@jkralik I've tested bundle with this change, seemed OK, is there a quick way to test if this doesn't break http-gw?

Reply:
This is fine - it is used only for build react app